lib/uart_blob/uart_blob_rx.c

```diff
@@ -0,0 +1,451 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <uart_blob_rx.h>
+#include <uart_blob_util.h>
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include <sys/__assert.h>
+#include <string.h>
+
+
+/*****************************************************************************
+* Structure definitions
+*****************************************************************************/
+
+struct fragment_out {
+	uint8_t *buf;
+	size_t idx;
+	size_t len;
+	size_t max_len;
+};
+
+
+/*****************************************************************************
+* Forward declarations 
+*****************************************************************************/
+
+static void cobs_user_cb(const struct uart_cobs_evt *const evt);
+
+
+/*****************************************************************************
+* Static variables
+*****************************************************************************/
+
+static bool done_pending		= false;
+static int rx_state			= OPCODE_NONE;
+static struct fragment_out fragment 	= {0};
```

Comment:
The code standard of zephyr dictates that static variables should not be explicitly assigned to 0, so just remove the initialization.